### ISSUE TITLE
508-defect-2 [COGNITION]: BAM2 Debt Letters - Debt detail table must have properly formatted heading row

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16769]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## User Story or Problem Statement

As a screen reader user, I want to understand the row/column relationships of the debt history table.

## Details

JAWS is not announcing the debt history table column headers. I think this is because the `<thead>` row needs TH tags with scope attributes. Screenshot and code snippet below.

## Acceptance Criteria

- [ ] Table header row HTML updated
- [ ] HTML validates
- [ ] No axe browser plugin violations
- [ ] No change in visual display for mobile, tablet, desktop breakpoints

## Environment

* https://staging.va.gov/manage-va-debt/your-debt/debt-detail
* Test user 1

## Solution (if known)

```diff
<table class="vads-u-margin-y--4">
  <thead>
    <tr>
-     <td class="vads-u-font-weight--bold">Date</td>
+     <th class="vads-u-font-weight--bold" scope="col">Date</th>
-     <td class="vads-u-font-weight--bold">Letter</td>
+     <th class="vads-u-font-weight--bold" scope="col">Letter</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        December 19, 2014
      </td>
      <td>
        <p class="vads-u-font-weight--bold vads-u-margin-bottom--0"></p>
        <p class="vads-u-margin-top--0">
          Account returned from Treasury Cross Servicing. Account is at TOP. TOP offsets will be
          applied to account as Federal funds become available.
        </p>
      </td>
    </tr>
  </tbody>
</table>
```

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs

![Screen Shot 2020-12-01 at 3 45 07 PM](https://user-images.githubusercontent.com/934879/100801043-0f2ff900-33ed-11eb-8e97-e8bb75346f05.png)

**Comment by bradl3yC** [comment]
@1Copenut this is ready for validation

**Comment by 1Copenut** [comment]
LGTM! Moving this one to close.
